src/meta/src/stream/source_manager.rs

```diff
@@ -109,6 +108,8 @@ impl<P: SourceProperties> ConnectorSourceWorker<P> {
         Ok(())
     }
 
+    /// On creation, connection to the external source service will be established, but `splits`
+    /// will not be updated until `tick` is called.
```

Comment:
What about make these ``` `` ```s ```[``]``` if possible so that they can be clicked? Also for many occurrences below.

Reply:
methods cannot be clicked (with RA) now

Reply:
https://github.com/rust-lang/rust-analyzer/issues/8487

Reply:
Since all code are in the single file, and methods are short (we can jump in the code body), I think it's not a big deal. 😟